apps/marketplace/components/SellerDashboard/SellerDashboard.scss

```diff
@@ -115,6 +125,28 @@ table.resultListing td, table.resultListing th {
   margin-top: -2.5em;
 }
 
+.notification {
+  position: relative;
+  margin-left: 5px;
+}
+.notification .circle {
+  border-radius: 50%;
+  border: solid 2px red;
+  padding: 1px 12px;
+  background-color: red;
+  display: inline;
+
+}
+.notification .count {
+  color: white;
+  width: 24px;
+  font-size: 1rem;
+  top: 0px;
+  left: 1px;
+  position: absolute;
+  text-align: center;
```

Comment:
The circle and count look a little off to me.  If you move `border-radius: 50%` from `circle` to `count` we should be able to delete the circle div from the markup.  Do you think that looks better (only for two digit numbers)?

Reply:
fixed! 👍